inc/inventorylog.class.php

```diff
@@ -0,0 +1,250 @@
+ * -------------------------------------------------------------------------
+ */
+
+if (!defined('GLPI_ROOT')) {
+    die("Sorry. You can't access directly to this file");
+}
+
+use Glpi\Application\View\TemplateRenderer;
+
+class PluginSccmInventoryLog extends \CommonDBTM
+{
+
+    public $dohistory = true;
+    public static $rightname = 'config';
+
+    public const SCCM_STATE_DONE = "sccm-done";
+    public const SCCM_STATE_FAIL = "sccm-fail";
+
+    static function install(Migration $migration)
+    {
+        global $DB;
+
+        $table = PluginSccmInventoryLog::getTable();
+
+        if (!$DB->tableExists($table)) {
+            $migration->displayMessage("Installing $table");
+            $query = "CREATE TABLE $table (
+                `id` int(11) NOT NULL AUTO_INCREMENT,
+                `name` varchar(255) NOT NULL,
+                `computers_id` int(11) unsigned DEFAULT NULL,
+                `error` text DEFAULT NULL,
+                `state` varchar(15) NOT NULL DEFAULT '0',
+                `date_mod` TIMESTAMP DEFAULT NULL,
+                PRIMARY KEY (`id`)
+            ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;";
+
+            $DB->queryOrDie($query, $DB->error());
+            $migration->updateDisplayPrefs([PluginSccmInventoryLog::class => [4, 5, 6, 7, 8]]);
+        }
+        $migration->migrationOneTable($table);
+
+        return true;
+    }
+
+
+    static function uninstall()
+    {
+        global $DB;
+
+        if ($DB->tableExists('glpi_plugin_sccm_inventorylogs')) {
+            $query = "DROP TABLE `glpi_plugin_sccm_inventorylogs`";
+            $DB->queryOrDie($query, $DB->error());
+        }
+        return true;
+    }
+
+    public static function canCreate()
+    {
+        return false;
+    }
+
+
+    public static function canPurge()
+    {
+        return false;
+    }
+
+    public static function canDelete()
+    {
+        return false;
+    }
+
+    public static function canUpdate()
+    {
+        return false;
+    }
+
+    public static function getTypeName($nb = 0)
+    {
+        return __('Inventory Logs', 'sccm');
+    }
+
+    public static function getMenuName()
+    {
+        return self::getTypeName();
+    }
+
+    public static function getIcon()
+    {
+        return "ti ti-file-search";
+    }
+
+    public static function showLogs()
+    {
+        $inventorylogs = new self();
+        $results = $inventorylogs->find();
+        TemplateRenderer::getInstance()->display(
+            '@sccm/inventorylogs.html.twig',
+            [
+                'results' => $results
+            ]
+        );
+
+        return true;
+    }
+
+    public static function getAllState(): array
+    {
+        return [
+            '' => '-------------',
+            self::SCCM_STATE_DONE   => __('Done'),
+            self::SCCM_STATE_FAIL   => __('Fail'),
+        ];
+    }
+
+    public static function getSpecificValueToDisplay($field, $values, array $options = [])
```

Comment:
You need to use 
```public static function getSpecificValueToDisplay($field, $values, array $options = [])``` too

To display a drodown on search form when "state" is selected